README.md

```diff
@@ -83,31 +89,31 @@ directory), or absolute.
 
 `Glob()` is meant to be a drop-in replacement for `filepath.Glob()`.
 
-## Patterns
+### Patterns
 
 **doublestar** supports the following special terms in the patterns:
 
-Special Terms | Meaning
-------------- | -------
-`*`           | matches any sequence of non-path-separators
-`**`          | matches any sequence of characters, including path separators
-`?`           | matches any single non-path-separator character
-`[class]`     | matches any single non-path-separator character against a class of characters ([see below](#character-classes))
-`{alt1,...}`  | matches a sequence of characters if one of the comma-separated alternatives matches
+| Special Terms | Meaning                                                                                                         |
```

Comment:
I prefer to keep markdown tables as simple as possible - they're much easier to maintain should changes need to be made.

Reply:
This is auto-fixed by editor every time i save the file. I will try to disable for this project before commit the change to avoid re-introduce the change.

I suggest you to use a markdown linter like this one in VS Code:
<https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint>

It identifies all markdown issues and make the final MD super-clean and reusable.